readalongs/align.py

```diff
@@ -413,7 +415,36 @@ def frames_to_time(frames):
         )
 
         split_silences(results["words"], final_end, dna_for_silence_splitting)
-
+    words_dict = {
+        x["id"]: {"start": x["start"], "end": x["end"]} for x in results["words"]
+    }
+    silence_offsets = defaultdict(int)
+    silence = 0
+    if results["tokenized"].xpath("//*[@silence]"):
+        endpoint = 0
+        for el in results["tokenized"].xpath("//*"):
+            if "silence" in el.attrib:
+                silence_ms = int(el.attrib["silence"])  # get silence (ms)
```

Comment:
I have a change request on this one. Can we use `parse_time()` from `readalongs.text.util`? That'll change your semantics, because the default unit will become the second, but anchors are in seconds by default, so I think that's appropriate, important even, for consistency.

Second reason for that ask is that Patrick requested I support the default format Audacity displays, which is `00h00m00.000s`. When I fix `parse_time()` to support that, it would be good for the silence syntax to benefit from the change too.

Reply:
By the way, it would be tempting to use an existing library for this, but I'd rather not introduce yet another dependency. It's not a big job to fix `parse_time()`, so I will do it.

Reply:
@roedoejet  Just pushed a change to `parse_time()` on this PR's branch, accepting h/m/s/ms. I was going to do it on its own PR, but then you'd have had to merge it into this one so I figured tacking it on to your PR made sense. I'll appreciate a review of those changes, if you don't mind.

Reply:
great, thanks for the quick addition! and yes, that makes sense to be consistent with the anchors feature.